src/Feliz.MaterialUI/Props.fs

```diff
@@ -219,7 +387,7 @@ type autocomplete =
   /// If `true`, the popup will ignore the blur event if the input is filled. You can inspect the popup markup with your browser tools. Consider this option when you need to customize the component.
   static member inline debug (value: bool) = Interop.mkAttr "debug" value
   /// The default input value. Use when the component is not controlled.
-  static member inline defaultValue (value: 'option []) = Interop.mkAttr "defaultValue" value
+  static member inline defaultValue (value: 'a) = Interop.mkAttr "defaultValue" value
```

Comment:
The changes to `defaultValue` and `value` shouldn't be there. The prop type check for this prop in the generator must probably be updated from `any | array` to `"props.multiple ? [] : null"`:

https://github.com/cmeeren/Feliz.MaterialUI/blob/6d1641d61655cf19a10400100b6348d322342c31/src/Feliz.Generator.MaterialUI/ApiParser.fs#L167

Would you additionally be able to investigate why the docs made this change? Does the change signal other capabilities/signatures?

Reply:
The only two autocomplete-related items were 

> [Autocomplete] Fix scroll reset after unselect the only option (#21280) @svikhristyuk
> [Autocomplete] Prevent default event for disabled options (#21390) @GregoryAndrievskiy

in the v4.10.2 release.

The docs now read: 

![image](https://user-images.githubusercontent.com/230547/87249369-026f3080-c45f-11ea-93c6-cf6fddf782aa.png)

Reply:
updated